kotlin/services/glue/src/main/kotlin/com/kotlin/glue/GlueScenario.kt

```diff
@@ -0,0 +1,295 @@
+//snippet-sourcedescription:[GlueScenario.kt demonstrates how to perform multiple AWS Glue operations.]
+//snippet-keyword:[AWS SDK for Kotlin]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[AWS Glue]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[03/17/2022]
+//snippet-sourceauthor:[scmacdon AWS]
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.kotlin.glue
+
+//snippet-start:[glue.kotlin.scenario.import]
+import aws.sdk.kotlin.services.glue.GlueClient
+import aws.sdk.kotlin.services.glue.model.DatabaseInput
+import aws.sdk.kotlin.services.glue.model.CreateDatabaseRequest
+import aws.sdk.kotlin.services.glue.model.StartJobRunRequest
+import aws.sdk.kotlin.services.glue.model.CreateJobRequest
+import aws.sdk.kotlin.services.glue.model.JobCommand
+import aws.sdk.kotlin.services.glue.model.CreateCrawlerRequest
+import aws.sdk.kotlin.services.glue.model.WorkerType
+import aws.sdk.kotlin.services.glue.model.GetDatabaseRequest
+import aws.sdk.kotlin.services.glue.model.GetTablesRequest
+import aws.sdk.kotlin.services.glue.model.S3Target
+import aws.sdk.kotlin.services.glue.model.GetCrawlerRequest
+import aws.sdk.kotlin.services.glue.model.GetJobsRequest
+import aws.sdk.kotlin.services.glue.model.GetJobRunsRequest
+import aws.sdk.kotlin.services.glue.model.DeleteJobRequest
+import aws.sdk.kotlin.services.glue.model.StartCrawlerRequest
+import aws.sdk.kotlin.services.glue.model.DeleteDatabaseRequest
+import aws.sdk.kotlin.services.glue.model.DeleteCrawlerRequest
+import aws.sdk.kotlin.services.glue.model.CrawlerTargets
+import java.util.concurrent.TimeUnit
+import kotlin.system.exitProcess
+//snippet-end:[glue.kotlin.scenario.import]
+
+/**
+To run this Kotlin code example, ensure that you have set up your development environment,
+including your credentials.
+
+For information, see this documentation topic:
+https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/setup.html
+ */
+
+//snippet-start:[glue.kotlin.scenario.main]
+suspend fun main(args:Array<String>) {
+
+    val usage = """
+        Usage:
+            <iam> <s3Path> <cron> <dbName> <crawlerName> <jobName> <scriptLocation> <locationUri>
+
+        Where:
+            iam - the ARN of the IAM role that has AWS Glue and S3 permissions.
+            s3Path - the Amazon Simple Storage Service (Amazon S3) target that contains data (for example, CSV data).
+            cron - a cron expression used to specify the schedule  (i.e., cron(15 12 * * ? *).
+            dbName - the database name. 
+            crawlerName - the name of the crawler. 
+            jobName - the name you assign to this job definition.
+            scriptLocation - specifies the Amazon S3 path to a script that runs a job.
+            locationUri - specifies the location of the database 

```

Comment:
locationUri - Specifies the location of the database.

Reply:
fixed